app/lib/efile/benefits_eligibility.rb

```diff
@@ -86,7 +86,11 @@ def claimed_recovery_rebate_credit
     private
 
     def rrc_eligible_filer_count
-      return intake.primary_tin_type == "ssn" ? 1 : 0 if tax_return.filing_status_single?
+      raise "unsupported filing type" unless tax_return.filing_status.in? %w[married_filing_jointly head_of_household single]
```

Comment:
It took me a while to understand this code. I think it'd be clearer with a `case` on the outside, like

```
case tax_return.filing status
when "head_of_household", "single"
  something
when "married_filing_jointly"
  something
else
  raise "unsupported filing type"
end
```

100% not a blocker, just a FYI.

Reply:
good call -- i can do that quickly :D